test cla verification

habeshaethiopia opened the pull request:
test the cla  verificaton

habeshaethiopia committed:
Update versioning in _version.py and pyproject.toml to reflect new development version

habeshaethiopia committed:
Bump version to 0.1.dev78 and update version scheme to post-release

habeshaethiopia committed:
Update version to 0.1.dev79 in _version.py and pyproject.toml

habeshaethiopia committed:
Bump version to 0.1.dev80 in _version.py and pyproject.toml

habeshaethiopia committed:
Bump version to 0.1.dev81 in _version.py

habeshaethiopia committed:
Bump version to 0.1.dev82 in _version.py and update version tuple

habeshaethiopia committed:
Trigger version increment

habeshaethiopia committed:
Bump version to 0.1.dev84 in _version.py and update version tuple

habeshaethiopia committed:
Bump version to 0.1.dev85 in _version.py and update version tuple

habeshaethiopia committed:
Bump version to 0.1.dev86 in _version.py and update version tuple

habeshaethiopia committed:
Merge branch 'main' of https://github.com/smorin/py-launch-blueprint

habeshaethiopia committed:
Bump version to 0.1.dev95 in _version.py and update version tuple

habeshaethiopia committed:
Bump version to 0.1.dev96 in _version.py and update version tuple; adjust local_scheme in pyproject.toml

habeshaethiopia committed:
Bump version to 0.1.dev97 in _version.py and update version tuple; uninstall py_launch_blueprint in Justfile

habeshaethiopia committed:
Remove uninstall command for py_launch_blueprint from Justfile

habeshaethiopia committed:
Replace uninstall command for py_launch_blueprint in Justfile with uv pip uninstall

habeshaethiopia committed:
Update version to 0.1.dev100 in _version.py; clean up Justfile and pyproject.toml

habeshaethiopia committed:
Bump version to 0.1.dev101 in _version.py; clean up pyproject.toml

habeshaethiopia committed:
Bump version to 0.1.dev102 in _version.py and update version tuple

habeshaethiopia committed:
No code changes made; empty commit.

habeshaethiopia committed:
Remove _version.py and update pyproject.toml to adjust versioning scheme

habeshaethiopia committed:
Clean up Justfile to remove various cache and temporary files; add _version.py generation

habeshaethiopia committed:
Update version to 0.1.dev106 and clean up Justfile to remove unnecessary cache removal commands

habeshaethiopia committed:
Update version to 0.1.dev106 and clean up Justfile to remove unnecessary cache removal commands

habeshaethiopia committed:
Remove _version.py and update fallback version in pyproject.toml

habeshaethiopia committed:
Fix formatting in pyproject.toml by adding newlines for better readability

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/habeshaethiopia/py-launch-blueprint?pullRequest=2) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/habeshaethiopia/py-launch-blueprint?pullRequest=2) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: smorin<br/>:x: habeshaethiopia<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/habeshaethiopia/py-launch-blueprint?pullRequest=2) it.</sub>

smorin committed:
Merge pull request #47 from habeshaethiopia/add-cla-integration

Add Contributor License Agreement (CLA) integration and automation

smorin committed:
Update Ruff linter configuration for test files

Add explanatory comments for ignored security-related linter rules in test files

smorin committed:
Added the first commit for sphinx based documentation.

smorin committed:
Update .readthedocs.yaml

fix comment

Co-authored-by: ellipsis-dev[bot] <65095814+ellipsis-dev[bot]@users.noreply.github.com>

smorin committed:
Merge pull request #48 from smorin/feature/readthedocsinit

Feature/readthedocsinit

habeshaethiopia committed:
Merge branch 'main' of https://github.com/smorin/py-launch-blueprint
